app/assets/javascripts/angular/controllers/static_pages.js.erb

```diff
@@ -15,4 +15,16 @@ app.controller('StaticPagesCtrl', ['$scope', '$http', '$auth', '$location', func
     });
   }
 
+  var orderBy = $filter('orderBy');
```

Comment:
You can put orderBy inside ng-repeat https://docs.angularjs.org/api/ng/filter/orderBy